semantic_release_template.yaml

```diff
@@ -0,0 +1,12 @@
+name: screwdriver-cd/semantic-release
+version: '1.0'
+description: Template for semantic release
+maintainer: idk
+config:
+  image: node:6
+  steps:
+    - check_prerequisite: |
+        if [ -z "$NPM_TOKEN" ]; then echo "NPM Token not set to NPM_TOKEN. Exiting..." && exit 1; fi
+        if [ -z "$GH_TOKEN" ]; then echo "Github Token not set to GH_TOKEN. Exiting..." && exit 1; fi
+    - install_binaries: npm i semantic-release@^7
```

Comment:
shouldn't we be using `--no-save` option so that `git` operations do not complain ?